src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -828,4 +829,18 @@ default FoodProperties getFoodProperties(ItemStack stack, @Nullable LivingEntity
         return self().getFoodProperties();
     }
 
+    /**
+     * Whether the given ItemStack should be excluded (if possible) when selecting the target hotbar slot of a "pick" action.
+     * By default, this returns true for enchanted stacks.
+     *
+     * @see Inventory#getSuitableHotbarSlot()
+     * @param player the player performing the picking
+     * @param inventorySlot the inventory slot of the item being up for replacement
+     * @return true to leave this stack in the hotbar if possible
+     */
+    default boolean isStickyInHotbar(ItemStack stack, Player player, int inventorySlot)
```

Comment:
This feels like a terrible name, not sure if this is any better but part of me feels like `isLowPriorityHotbarSlot` or something might be at least slightly better? (Still not quite sure if that is a decent name either though)

Reply:
I suggest isReplaceableByPick or isReplaceableByPickAction